src/main/java/net/openhft/chronicle/bytes/internal/BytesInternal.java

```diff
@@ -602,11 +609,11 @@ static void parseUtf82(@NotNull StreamingDataInput bytes, @NotNull Appendable ap
                     count += utf ? 2 : 1;
                     if (count > length)
                         throw new UTFDataFormatRuntimeException(
-                                "malformed input: partial character at end");
+                                MALFORMED_INPUT_PARTIAL_CHARACTER_AT_END);
                     int char2 = bytes.readUnsignedByte();
                     if ((char2 & 0xC0) != 0x80)
                         throw new UTFDataFormatRuntimeException(
-                                "malformed input around byte " + count + " was " + char2);
+                                MALFORMED_INPUT_AROUND_BYTE + count + WAS + char2);
```

Comment:
maybe should printf?

Reply:
Or format(..) in this case, the template string would make more sense as a constant than the two separate string